doc/locale/ja/LC_MESSAGES/install/migrate_from_ppa.po

```diff
@@ -0,0 +1,75 @@
+# Japanese translations for Groonga package.
```

Comment:
I will delete this duplicated file.

Reply:
fix: [c3a8f17](https://github.com/groonga/groonga/pull/2260/commits/c3a8f1794a8772b30197adb7473b9e8eb795b44a)